.nvmrc

```diff
@@ -1 +1 @@
-4
+8
```

Comment:
Seems like Node.JS version 8 is the baseline, so updating to indicate that here too.